source/support/FOBactions.sqf

```diff
@@ -3,6 +3,10 @@ _fob = _this select 0;
 
 _fob addaction ["<t color='#ff00ff'>Player stats</t>","dialog\info\info.sqf", "", 0, true, true, "", "_this == player"];
 _fob addaction ["<t color='#15ff00'>Request ammobox drop(2CP)</t>","support\fob_ammobox.sqf", "", 0, true, true, "", "_this == player"];
+if (support_armory_available) then {
+	_fob addaction ["<t color='#ff0066'>Armory 1 (VAS)</t>","VAS\open.sqf", "", 0, true, true, "", "_this == player"];
+	_fob addaction ["<t color='#ff0066'>Armory 2 (VA)</t>","bisArsenal.sqf", "", 0, true, true, "", "_this == player"];
+};
```

Comment:
Here we have tabs. I like to get away from tabs in source code so that if people have different tab stops they do not have the code come out funky looking when some of the source has spaces and other chunks of the source have tabs. I realize this is very nit picky but it goes a long way in making the code readable. In the future once all the tabs are removed from the source I plan on having a build process that will verify no tabs are in the code. Exceptions will have to be made for things like VAS, pFLIR and TAW but I feel it is very worth it.